packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/include/camera_avfoundation/messages.g.h

```diff
@@ -28,6 +28,23 @@ typedef NS_ENUM(NSUInteger, FCPPlatformCameraLensDirection) {
 - (instancetype)initWithValue:(FCPPlatformCameraLensDirection)value;
 @end
 
+typedef NS_ENUM(NSUInteger, FCPPlatformCameraLensType) {
+  /// A built-in wide-angle camera device type.
+  FCPPlatformCameraLensTypeWide = 0,
+  /// A built-in camera device type with a shorter focal length than a wide-angle camera.
```

Comment:
isn't telephoto *longer* focal length than wide angle?

Reply:
You're right, let me update the doc